### PR TITLE
ssg/constants.py: fix the alinux2/3 full name error

### LINKED ISSUE #9267: ssg/constants.py: fix the alinux2/3 full name error
#### Description of problem: The full name of alinux2/3 is Alibaba Cloud Linux 2/3 rather than Alinux 2/3, so
fix it. Besides, fixing the full name error of alinux3 can resolve some
notchecked problems when using oscap.

### PR DESCRIPTION
**Pull request by hustliyilin**
#### Description:

- The full name of alinux2/3 is Alibaba Cloud Linux 2/3 rather than Alinux 2/3, so
fix it. Besides, fixing the full name error of alinux3 can resolve some
notchecked problems when using oscap.

- Fixes #9267

### CODE REVIEW

File: ssg/constants.py [hunk 1]
```diff
@@ -193,8 +193,8 @@
 }
 
 FULL_NAME_TO_PRODUCT_MAPPING = {
-    "Alinux 2": "alinux2",
-    "Alinux 3": "alinux3",
```
Comment (jan-cerny) [inline ssg/constants.py:197]: There are multiple occurences of this name is  `CMakeLists.txt` and `constants.xslt`, please fix them as well.

Comment (hustliyilin) [reply ssg/constants.py:197]: > constants.xslt

Hi @jan-cerny ，

Thanks for your apply. However
- In [Alinux2/constants.xslt](https://github.com/ComplianceAsCode/content/blob/master/products/alinux2/transforms/constants.xslt#L5) and [Alinux3/constants.xslt](https://github.com/ComplianceAsCode/content/blob/master/products/alinux3/transforms/constants.xslt#L5) I already fixed them.
- In [CMakeLists.txt](https://github.com/ComplianceAsCode/content/blob/master/CMakeLists.txt) , it doesn't include the full name of the product.

Thanks again. If you have any questions, please let me know.

Comment (Mab879) [reply ssg/constants.py:197]: These might be the place that @jan-cerny is talking about in [CMakeLists.txt](https://github.com/ComplianceAsCode/content/blob/master/CMakeLists.txt)

ALinux2
* https://github.com/ComplianceAsCode/content/blob/master/CMakeLists.txt#L69
* https://github.com/ComplianceAsCode/content/blob/master/CMakeLists.txt#L263

ALinux3
* https://github.com/ComplianceAsCode/content/blob/master/CMakeLists.txt#L70
* https://github.com/ComplianceAsCode/content/blob/master/CMakeLists.txt#L264

It appears we are inconstant with short name vs. long name in this file. @jan-cerny thoughts?

Comment (hustliyilin) [reply ssg/constants.py:197]: Hi @Mab879 , 

Thanks for your reply. Now I fix all inconstant places. @jan-cerny

**Review by jan-cerny — APPROVED** [review]

**Pull request merged**